### DISCUSSION TITLE
Error: Unicorn mode doesn't seem to work

### DISCUSSION THREAD

**Discussion by YuriiCrimson** [AFLplusplus/AFLplusplus#1360] [Q&A]
I cloned afl++ and started compiling it completely. but i take error
Terminal:
Installed /home/scarybear/.local/lib/python3.9/site-packages/unicornafl-2.0.1-py3.9.egg
Processing dependencies for unicornafl==2.0.1
Searching for unicorn==2.0.0rc6
Best match: unicorn 2.0.0rc6
Processing unicorn-2.0.0rc6-py3.9-linux-x86_64.egg
unicorn 2.0.0rc6 is already the active version in easy-install.pth

Using /home/scarybear/.local/lib/python3.9/site-packages/unicorn-2.0.0rc6-py3.9-linux-x86_64.egg
Finished processing dependencies for unicornafl==2.0.1
[*] If needed, you can (re)install the bindings from `./unicornafl/bindings/python` using `python setup.py install`
[*] Unicornafl bindings installed successfully.
[*] Testing unicornafl python functionality by running a sample test harness
Showmap
[-] Error: Unicorn mode doesn't seem to work!
make: [GNUmakefile:627: distrib] Error 1 (ignored)

**Comment by vanhauser-thc** [comment]
can you try the currrent dev branch please?

**Reply by YuriiCrimson** [reply]
ye, test passed, thanks

**Comment by dukptkey** [comment]
I had the same error with the master branch, and it was fixed by manually installing `libpython3`.   

In my case, the command was `sudo apt install libpython3.11-dev libpython3.11`
